### PR TITLE
AG-8238 Bugfix ErrorBar Visibility Toggle

### PR DESCRIPTION
**Pull request by olegat**
https://ag-grid.atlassian.net/browse/AG-8238
https://ag-grid.atlassian.net/browse/AG-9014

Keeping the ErrorBarNode instance in the Series.contentGroup breaks the
rendering when an item is highlighted.

Keeping the ErrorBarNode instance in the Series.highlightGroup fixes this
rendering bug when an item is highlighted but breaks visibility toggling.

Therefore, we need to move the ErrorBarNode back and forth between the
contentGroup and highlightGroup when the highlighting changes.

### CODE REVIEW

**Review by alantreadway — COMMENTED** [review]
Discussed offline.

File: packages/ag-charts-community/src/chart/layers.ts [hunk 1]
```diff
@@ -7,7 +7,6 @@ export enum Layers {
     AXIS_ZINDEX = 20,
     SERIES_CROSSLINE_RANGE_ZINDEX = 30,
     SERIES_LAYER_ZINDEX = 500,
-    SERIES_ERRORBAR_ZINDEX = 600,
```
Comment (olegat) [inline packages/ag-charts-community/src/chart/layers.ts:10]: Note: this is no longer necessary now that the error bar are in their own group.

**Comment by olegat** [comment]
> Discussed offline.

I've added an `annotationGroup` to _series.ts_ as we've discussed.

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**